hack/local-up-cluster.sh

```diff
@@ -688,111 +687,113 @@ function start_kubelet {
 
     priv_arg=""
     if [[ -n "${ALLOW_PRIVILEGED}" ]]; then
-      priv_arg="--allow-privileged=${ALLOW_PRIVILEGED} "
+      priv_arg="--allow-privileged=${ALLOW_PRIVILEGED}"
```

Comment:
To make the following fixes work correctly:

In ./hack/local-up-cluster.sh line 753:
      ${priv_arg}
      ^---------^ SC2206: Quote to prevent word splitting, or split robustly with mapfile or read -a.